client/src/hooks/useUser.js

```diff
@@ -47,6 +47,50 @@ export const useUser = (defaultUser, defaultToken) => {
     setToken()
   }
 
+  const updateEmail = async (email) => {
+    const updateRequest = await api.user.update(user.id, { email }, token)
+    if (updateRequest.isOk) refreshUser()
+    return updateRequest.isOk
+  }
+
+  const getTeam = (teamOrId) => {
+    const teamId = teamOrId.id || teamOrId
+    if (user.organizations) {
+      return user.organizations.find((t) => t.id === teamId)
+    }
+
+    return teamId
+  }
+
+  const isPersonalResource = (resource) => {
+    if (typeof resource.organization === 'number') {
+      return resource.organization === user.personal_organization.id
+    }
+    if (typeof resource.organization.id === 'object') {
+      return resource.organization.id === user.personal_organization.id
+    }
+
+    // there was no way to tell
+    // since no org was defined
```

Comment:
Is this referring to the personal org not being created? That should be fixed now.

Reply:
it should always work but when sentry is added this should be sent there